torchvision/prototype/transforms/__init__.py

```diff
@@ -3,7 +3,7 @@
 from ._transform import Transform  # usort: skip
 
 from ._augment import RandomCutmix, RandomErasing, RandomMixup
-from ._auto_augment import AugMix, AutoAugment, RandAugment, TrivialAugmentWide
+from ._auto_augment import AugMix, AutoAugment, AutoAugmentPolicy, RandAugment, TrivialAugmentWide
```

Comment:
If this is missing in `main`, please add this in a separate PR

Reply:
Yes please, let's fix this elsewhere. It's likely we won't actually merge this PR but rather use it only for training.

Reply:
#6411.